src/main/java/ca/warp7/frc2022/subsystems/Launcher.java

```diff
@@ -57,11 +72,10 @@ public void calcOutput() {
         if (targetRPS == 0.0)
             this.setVoltage(0.0);
         else
-            this.setVoltage((targetRPS + getError() * kLauncherKp) * kLauncherKv +
-                    kLauncherKs * Math.signum(targetRPS));
+            this.setVoltage(voltageCalculator.calculate(this.getError()));
     }
 
     private void setVoltage(double voltage) {
-        //flywheelMasterNeo.set(voltage / 12);
+        launcherMotorMaster.set(ControlMode.Position, voltage / 12);
```

Comment:
This should be `PercentOutput`, not `Position`. Position control doesn't make sense with the flywheel.